schema/request-representation.yaml

```diff
@@ -0,0 +1,58 @@
+$schema: http://json-schema.org/draft-04/schema
+title: Panda Request Representation
+description: >
+  A standard format for expressing all of the dimensions of an HTTP request in
+  a single, nested object.  Usable in AWS Lambda handler functions, or in any
+  simiar function-based approach.
+
+example:
+
+  url:
+    path:
+      account: wyrdspirits
```

Comment:
Ah, I see—this expresses what the application developer can expect in their handler?